src/Init/NotationExtra.lean

```diff
@@ -337,14 +337,15 @@ macro_rules
 
 section
 open Lean.Parser.Tactic
+syntax cdotTk := patternIgnore("·" <|> ".")
```

Comment:
This is the only case where I had to introduce a new syntax abbrev, since a token quotation inside `patternIgnore` obviously can't work anymore. And yes, I went with a less "cute" name than `nomatch` after all.